ingest-app/src/cmr/ingest/system.clj

```diff
@@ -97,6 +98,10 @@
                                 (af/refresh-acl-cache-job "ingest-acl-cache-refresh")
                                 jvm-info/log-jvm-statistics-job))
               :caches {acl/token-imp-cache-key (acl/create-token-imp-cache)
+                       ;; Caches a map of tokens to the security identifiers
+                       context-augmenter/token-sid-cache-name (context-augmenter/create-token-sid-cache)
```

Comment:
Hrm; the way this was written (this work was originally done in the search app), if the sid caching isn't present, the acl lookups fail (I tried running the tests with the sid caching removed from the ingest system -- a no-go as it is currently implemented).

I'll look into this now to see how search and ingest can both use this code (or parts of it) ...